.github/workflows/CI.yaml

```diff
@@ -147,15 +166,48 @@ jobs:
 
       - name: "Test: Gear workspace"
         run: ./scripts/gear.sh test gear --release
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: JS metadata"
         run: ./scripts/gear.sh test js
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+
+      - name: "Test: Process gear-test"
+        run: ./scripts/gear.sh test gtest
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
```

Comment:
what's the reason to keep flag here?
https://github.com/gear-tech/gear/pull/932/files#r872209943